app/views/course/assessment/submission/submissions/_buttons.html.slim

```diff
@@ -0,0 +1,23 @@
+- submission = f.object
+div.submission-buttons
+  - if defined?(save) && save
```

Comment:
They added it back for rails 5. I'm not sure if it's private or public API now though, but definitely something that has been requested before (even by [DHH](https://github.com/rails/rails/issues/18962)), so I don't think they will remove it unless they deprecate it properly. 

References: 
 - https://github.com/rails/rails/pull/18970
 - http://api.rubyonrails.org/classes/ActionView/Base.html